js-api-spec/legacy/importer.test.ts

```diff
@@ -924,3 +925,34 @@ describe('when importer returns non-string contents', () => {
     );
   });
 });
+
+it('compiles multiple nested relative imports and incorrect load path with renderSync', () => {
+  sandbox(dir => {
+    dir.write({
+      'src/index.scss': `
+        @import "./foo/_a.scss";
+        @import "./_includes.scss";
+      `,
+      'include/foo/_a.scss': '/* A */',
+      'include/foo/_b.scss': '/* B */',
+      'src/_includes.scss': `
+        @import "./foo/_a.scss";
+        @import "./foo/_b.scss";
+      `,
+    });
+
+    const result = sass.renderSync({
+      file: dir('src/index.scss'),
+      importer: function (url) {
+        const path = dir('./include', url);
```

Comment:
This IIUC normalizes the path so it fetches for files under `./include`

but because `includes.scss` is loaded from a different directory then I think this messes up with `renderSync`

And I wonder if it's worth fixing it or whether this can be considered to be a reasonable error because of the use of a deprecated API 🤔